website/docs/r/kubernetes_cluster_node_pool.html.markdown

```diff
@@ -145,6 +149,104 @@ If `enable_auto_scaling` is set to `false`, then the following fields can also b
 
 ---
 
+A `kubelet_config` block supports the following:
+
+* `allowed_unsafe_sysctls` - (Optional) Specifies the allow list of unsafe sysctls command or patterns (ending in `*`). Changing this forces a new resource to be created.
+
+* `container_log_max_line` - (Optional) Specifies the maximum number of container log files that can be present for a container. must be at least 2. Changing this forces a new resource to be created.
+
+* `container_log_max_size_mb` - (Optional) Specifies the maximum size (e.g. 10Mi) of container log file before it is rotated. Changing this forces a new resource to be created.
+
+* `cpu_cfs_quota_enabled` - (Optional) Is CPU CFS quota enforcement for containers enabled? Changing this forces a new resource to be created.
+
+* `cpu_cfs_quota_period` - (Optional) Specifies the CPU CFS quota period value. Changing this forces a new resource to be created.
+
+* `cpu_manager_policy` - (Optional) Specifies the CPU Manager policy to use. Possible values are `none` and `static`, Changing this forces a new resource to be created.
+
+* `image_gc_high_threshold` - (Optional) Specifies the percent of disk usage after which image garbage collection is always run. Must be between `0` and `100`. Changing this forces a new resource to be created.
+
+* `image_gc_low_threshold` - (Optional) Specifies the percent of disk usage before which image garbage collection is never run. Must be between `0` and `100`. Changing this forces a new resource to be created.
```

Comment:
this description doesn't make sense?

Reply:
https://kubernetes.io/docs/concepts/cluster-administration/kubelet-garbage-collection/#image-collection

Disk usage above the high threshold will trigger garbage collection. The garbage collection will delete least recently used images until the low threshold has been met